ruby/vampires.rb

```diff
@@ -62,31 +62,18 @@
 	end
 
 	case 
-		when applicant_real_age == true && 
-			if garlicbread_preference == true || health_insurance == true
-				puts "Probably not a vampire."
-			elsif applicant_name == "Drake Cula" || applicant_name == "Tu Fang"
-				puts "Definitely a vampire."
-			elsif applicant_allergy == "sunshine" 
-				puts "Probably a vampire."
-			else
-			puts "Results inconclusive"
-			end
-				
-		#when applicant_allergy == "sunshine"
-			#puts "Probably a vampire."
-		#when applicant_name == "Drake Cula" || applicant_name == "Tu Fang"
-			#puts "Definitely a vampire."
-		when applicant_real_age == false && 
-			if garlicbread_preference == false || health_insurance == false
-				puts "Probably a vampire."
-			elsif garlicbread_preference == false && health_insurance == false
-				puts "Almost certainly a vampire."
-			else
-				puts "Results inconclusive"
-			end
-		else
-				puts "Results inconclusive"
+		when applicant_allergy == "sunshine"
```

Comment:
Instead of using a case or an if/else tree, you could define each condition on its own separate if statement. Otherwise the case / if tree will stop once it finds the first true condition. 

Hope this helps!